ARTwarp_csv.m

```diff
@@ -4,7 +4,9 @@
 
 global vigilance bias learningRate maxNumCategories maxNumIterations sampleInterval resample callback2
 
-load ARTwarp.mat
+load Default.mat
+
+addpath(pwd);
```

Comment:
Same change as @tristankleyn did in #10. Would it make sense to check other files?